src/styles/main.scss

```diff
@@ -19,6 +19,13 @@ body {
   font-family: Arial, Helvetica, sans-serif;
 }
 
+.centered {
```

Comment:
It's still not centered
How does `position: absolute` work?

![image](https://github.com/mate-academy/layout_calendar/assets/86923089/2275fa1a-ac49-4ed4-aeb9-314a2d70ab6c)

Hint: you don't need position: absolute here, you may use margin. Also, additional wrapper is redundant - add styles to body (but using class selector)

Reply:
This comment still hasn't been fixed yet - please fix it. If you have any questions - feel free to ask in the chat